### PR TITLE
Fixed console.log logic Sec3 Lesson 4 (JSDAO)

### PR DESCRIPTION
**Pull request by TrinidadHype**
Problem: Currently in Section 3 Lesson 4 where we are setting up our voteModule, there is a console.log that prints '🥵 User has already voted' whether a user has actaully voted or not. 

Expected Behavior: The intended behaviour is to have it only print if the user already voted.

Solution: I have changed the logic by adding an if statement to check if the user voted and only print when the user has actually voted.

### CODE REVIEW

**Review by nachoiacovino — APPROVED** [review]

File: JS_DAO/en/Section_3/Lesson_4_Let_Users_Vote_On_Proposals.md [hunk 1]
```diff
@@ -162,7 +162,9 @@ useEffect(() => {
     .hasVoted(proposals[0].proposalId, address)
     .then((hasVoted) => {
       setHasVoted(hasVoted);
-      console.log("🥵 User has already voted")
+      if(hasVoted) {
```
Comment (adilanchian) [inline JS_DAO/en/Section_3/Lesson_4_Let_Users_Vote_On_Proposals.md:165]: Can you add a space between `if` and `(hasVoted)` ?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Really good catch!

Can you make one small change for me please :)?

File: JS_DAO/en/Section_3/Lesson_4_Let_Users_Vote_On_Proposals.md (hunk 1 above)
Comment (TrinidadHype) [reply JS_DAO/en/Section_3/Lesson_4_Let_Users_Vote_On_Proposals.md]: Submitted the change, it should now have the space between `if` and `(hasVoted)` :)

**Comment by adilanchian** [comment]
Cheers @TrinidadHype !

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR VERY FIRST GITHUB CONTRIBUTION!!

👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏

**Comment by adilanchian** [comment]
@all-contributors please add @TrinidadHype  for doc

**Pull request merged**